ipaserver/plugins/caacl.py

```diff
@@ -17,6 +16,11 @@
 from ipalib import _, ngettext
 from ipapython.dn import DN
 
+try:
+    import pyhbac
+except ImportError:
+    pyhbac = None
```

Comment:
`SkipPluginModule` should be raised here as well. Alternatively, you could move `acl_evaluate` and this import to cert.py, since that the only place where it's used, and raise `SkipPluginModule` there.